### PR TITLE
Add the collaborate buttons/links and remove the blog link from the h…

### PR DESCRIPTION
**Pull request by schmidtw**
…eader.

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -14,30 +14,44 @@ <h1>Units of scale</h1>
...
+    <div class="col-md-4 col-sm-6 col-xs-12 feature-item">
       <a href="/docs/wrp/overview/">
         <h2><i class="fas fa-envelope"></i> WRP</h2>
         <p>The Web Routing Protocol provides a cross transport data envelope.</p>
       </a>
     </div>
+    <div class="col-md-4 col-sm-6 col-xs-12 feature-item">
+      <a href="https://discussion.xmidt.io">
+				<h2><i class="fas fa-comments"></i> Collaborate</h2>
+        <p>We've added a place for discussing goals, issues and related interests.</p>
```
Comment (kristinaspring) [inline content/index.html]: "We've added" sounds good now, but in a few months when it isn't so new are we going to change this? "Added" kind of implies that it just happened.  Can we get rid of the first two words? Or, say something welcoming like "Come talk with us about XMiDT's goals, issues, and related interests!"

File: content/index.html [hunk 2]
```diff
@@ -14,30 +14,44 @@ <h1>Units of scale</h1>
 
 <div class="container">
   <div class="row">
-    <div class="col-md-3 col-sm-6 col-xs-12 feature-item">
+    <div class="col-md-4 col-sm-6 col-xs-12 feature-item">
```
Comment (kristinaspring) [inline content/index.html]: How did you decide to change this to col-md-4? What does this mean?

File: content/docs/community/overview.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Overview
+sort_rank: 10
+---
+
+# Community
+Xmidt and it's ecosystem is developed in the open.  Here are some of the
```
Comment (kristinaspring) [inline content/docs/community/overview.md]:
```suggestion
XMiDT and its ecosystem is developed in the open.  Here are some of the 
```

File: content/docs/community/overview.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+sort_rank: 10
+---
+
+# Community
+Xmidt and it's ecosystem is developed in the open.  Here are some of the 
+channels we use to communicate and contribute:
+
+
+## Issue Tracker
+Use the GitHub issue tracker for the various [Xmidt repositories](http://github.com/xmidt-org)
```
Comment (kristinaspring) [inline content/docs/community/overview.md]:
```suggestion
Use the GitHub issue tracker for the various [XMiDT repositories](http://github.com/xmidt-org)
```

File: content/docs/community/overview.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+to the [discussion site](https://discussion.xmidt.io) rather than filing a GitHub issue.
+
+*Please do not ask individual project members for support. Use the channels
+above instead, where the whole community can help you and benefit from the
+solutions provided.*
+
+## Contributing
+
+We welcome community contributions! Please see the `CONTRIBUTING.md` file in
+the respective Xmidt repository for instructions on how to submit changes. If
```
Comment (kristinaspring) [inline content/docs/community/overview.md]:
```suggestion
the respective XMiDT repository for instructions on how to submit changes. If
```

File: content/docs/community/overview.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+you are planning on making more elaborate or controversial changes, please
+discuss them on the [discussion site](https://discussion.xmidt.io) before
+sending a pull request.
+
+## Discussion
+
+An important part of the community is discussing design proposals, pull requests,
+operational experiences and ideas for making the projects better.  We have added
```
Comment (kristinaspring) [inline content/docs/community/overview.md]:
```suggestion
operational experiences and ideas for making the projects better.  We have
```

File: content/index.html [hunk 7]
```diff
@@ -14,30 +14,44 @@ <h1>Units of scale</h1>
...
       <a href="/docs/wrp/overview/">
-        <h2><i class="fas fa-envelope"></i> WRP</h2>
+        <h2><i class="fas fa-envelope-open-text"></i> WRP</h2>
         <p>The Web Routing Protocol provides a cross transport data envelope.</p>
       </a>
     </div>
+    <div class="col-md-4 col-sm-6 col-xs-12 feature-item">
+      <a href="https://discussion.xmidt.io">
+				<h2><i class="fas fa-comments"></i> Collaborate</h2>
+        <p>We've added a place for discussing goals, issues and related interests.</p>
```
Comment (kristinaspring) [inline content/index.html]:
```suggestion
        <p>A place for discussing goals, issues and related interests.</p>
```

Comment (schmidtw) [reply content/index.html]: Changed.  Good suggestion!

File: content/docs/community/overview.md (hunk 5 above)
Comment (schmidtw) [reply content/docs/community/overview.md]: Changed.  Good suggestion!

File: content/docs/community/overview.md (hunk 4 above)
Comment (schmidtw) [reply content/docs/community/overview.md]: Changed.  Good suggestion!

File: content/docs/community/overview.md (hunk 3 above)
Comment (schmidtw) [reply content/docs/community/overview.md]: Changed.  Good suggestion!

File: content/index.html (hunk 1 above)
Comment (schmidtw) [reply content/index.html]: I missed the "We've added" part.  Thank you for suggesting something better below.

**Pull request merged**